client/src/featureform/serving.py

```diff
@@ -445,6 +445,33 @@ def to_numpy(self):
     def __repr__(self):
         return "Features: {} , Label: {}".format(self.features(), self.label())
 
+class BatchRow:
+
+    def __init__(self, rows=None):
+        self._features = []
+        self._labels = []
+        if rows is None:
+            rows = []
+        self._rows = rows
+        for row in rows:
+            self.append(row)
+
+    def append(self, row):
+        self._features.append(row.features())
+        self._labels.append(row.label())
+        self._rows.append(row)
+
+    def features(self):
+        return self._features
+
+    def labels(self):
+        return self._labels
+
+    def to_list(self):
+        return self._rows
+
+    def len(self):
```

Comment:
would recommend using 
```
def __len__(self):
      return len(self._rows)
```
because the user would expect to use `len(rows)`. 
If you agree and we change this, then line 364 can be reverted back to `len(rows)`.